### ISSUE TITLE
Mapping Exception

### ISSUE THREAD

**Issue by AdamJHall** [oroinc/OroCRMDotmailerBundle#3]
Hello,

While doing some custom work on CRM I kept running into 

```
The class 'Oro\Bundle\DotmailerBundle\Entity\ChangedFieldLog' was not found in the chain configured namespaces Akeneo\Bundle\BatchBundle\Entity
```

The stack trace shows that it's coming from Oro\Bundle\DotmailerBundle\EventListener\EntityUpdateListener:106. While talking to Sergey on slack the other week (about a similar issue) he mentioned that entity listeners should not use the entity manager provided by the $args and instead one from the registry. So simple fix is to change:

Oro\Bundle\DotmailerBundle\EventListener\EntityUpdateListener:postFlush:106
```
$em = $args->getEntityManager();
$repository = $em->getRepository(ChangedFieldLog::class);
```

to 

```
$repository = $this->doctrineHelper->getEntityRepository(ChangedFieldLog::class);
```

I've tested locally and this fixes it on my environment.

**Comment by d-beekeeper** [comment]
Hi AdamJHall

Thanks for notice! You are absolutely right. Oro\Bundle\EntityBundle\ORM\DoctrineHelper must be used inside doctrine listeners instead of entityManager.

I made a pull request into our repository, so this fix should appear in closest releases.

Internal ticket: CRM-8756

**Comment by mbessolov** [comment]
Fixed by 943d6a62f6072f8942f1da45244e409fe161c741
